src/ActionList/Item.tsx

```diff
@@ -13,13 +13,18 @@ export interface ItemProps extends React.ComponentPropsWithoutRef<'div'>, SxProp
   /**
    * Primary text which names an `Item`.
    */
-  text: string
+  text?: string
 
   /**
    * Secondary text which provides additional information about an `Item`.
    */
   description?: string
 
+  /**
+   * Custom children content to show if text/description is not flexible enough.  If supplied, leading/traling elements can still be used.
+   */
+  children?: React.ReactNode
```

Comment:
Is `children` implicitly part of every component’s props contract already?

Reply:
Indeed it is!  Removed the explicit prop ✅